python/runner/runner.py

```diff
@@ -186,11 +187,20 @@ async def run_instance(self, config, args):
 
         self.logger.info(f'Sending PANG')
         monitoring = self.streams[CC.MONITORING]
-        pang_provides_data = {
+        pang_data = {
             'provides': '',
             'contentType': ''
         }
-        send_encoded_msg(monitoring, msg_codes.PANG, pang_provides_data)
+        send_encoded_msg(monitoring, msg_codes.PANG, pang_data)
```

Comment:
Do we need to send empty PANG at all? This looks like a leftover from previous version of the code ;)